tests/builder.rs

```diff
@@ -281,6 +285,28 @@ impl<'a> Builder<'a> {
             )
             .as_str(),
             false,
+            b"",
+        ).unwrap();
+    }
+
+    pub fn build_empty_file_with_prefetch(&mut self) {
```

Comment:
Why not follow `build_lower` and `build_upper` (create new one named `builder_empty`)? so that we can reuse the `test` func in `tests/smoke.rs`.

Reply:
Bugs only occur when prefetched and empty files are combined, so there are no build_lower and build_upper used here

Reply:
This case and other tests have now been combined.